graphics/tinygl/zline.cpp

```diff
@@ -105,6 +105,8 @@ void FrameBuffer::drawLine(const ZBufferPoint *p1, const ZBufferPoint *p2) {
 	int sr, sg, sb;
 
 	if (kInterpZ) {
+		if (!n)
```

Comment:
you need to drop this commit, it's already in our repository

Reply:
Dropped, but now GitHub is saying "This branch cannot be rebased due to too many changes" so I hope I did not killed accidentally :cry: